### PR TITLE
Comelit SimpleHome integration doc

### PR DESCRIPTION
**Pull request by chemelli74**
## Proposed change

Introduce documentation for Comelit integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/96552
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/comelit.markdown [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Comelit SimpleHome
+description: Instructions on how to integrate Comelit SimpleHome home automation devices into Home Assistant.
+ha_category:
+  - Light
+ha_release: '2023.09'
```
Comment (bieniu) [inline source/_integrations/comelit.markdown]: I think that `2023.9` is a default format here.

Comment (chemelli74) [reply source/_integrations/comelit.markdown]: Thx, fixed

**Review by bieniu — APPROVED** [review]
Looks good 👍

**Comment by chemelli74** [comment]
> Looks good 👍

Thx !

Simone

**Pull request merged**